src/middle/Stan_math_signatures.ml

```diff
@@ -140,8 +140,26 @@ let mk_declarative_sig (fnkinds, name, args) =
 let full_lpdf = [Lpdf; Rng; Ccdf; Cdf]
 let full_lpmf = [Lpmf; Rng; Ccdf; Cdf]
 let reduce_sum_functions = ["reduce_sum"; "reduce_sum_static"]
+
+let variadic_ode_functions =
```

Comment:
tiny nitpick - these might be slightly faster as a set, since the only operation seems to be checking membership

Reply:
Thanks. Will move to set!